qiskit/visualization/state_visualization.py

```diff
@@ -18,11 +18,14 @@
 Visualization functions for quantum states.
 """
 
+from typing import Optional, List, Union
 from functools import reduce
 import colorsys
 import numpy as np
+import sympy
```

Comment:
Can this be moved to a runtime import? (sympy can take a fair amount of time to import, so we try to keep it out of the set of dependencies that are pulled in when someone runs `import qiskit`.)